estate/models/estate_property.py

```diff
@@ -0,0 +1,41 @@
+from odoo import fields, models
+import datetime
+from dateutil.relativedelta import relativedelta
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "this is the estate property model"
+    name = fields.Char('Title', required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date('Available From', default=datetime.date.today() + relativedelta(months=5), copy=False)
+    expected_price = fields.Float(digits=(20, 2), required=True)
+    selling_price = fields.Float(digits=(20, 2), readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer('Living Area (sqm)')
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer('Garden Area (sqm)')
+    garden_orientation = fields.Selection(selection=[
+        ('north', 'North'),
+        ('south', 'South'),
+        ('east', 'East'),
+        ('west', 'West')])
```

Comment:
```suggestion
    garden_orientation = fields.Selection(
        selection=[
            ('north', 'North'),
            ('south', 'South'),
            ('east', 'East'),
            ('west', 'West')
        ]
    )
```
just styling preference, and like this it gives more space to your code, which makes it more readable